com.github.calo001.luna.json

```diff
@@ -0,0 +1,50 @@
+{
+    "app-id" : "com.github.calo001.luna",
+    "base" : "io.elementary.BaseApp",
+    "base-version" : "juno",
+    "runtime" : "org.gnome.Platform",
+    "runtime-version" : "3.30",
+    "sdk" : "org.gnome.Sdk",
+    "command" : "com.github.calo001.luna",
+    "finish-args" : [
+        "--share=ipc", "--socket=x11",
+        "--socket=wayland",
```

Comment:
Does it really work with Wayland?

Reply:
I'm not sure, I was reviewing some similar projects, I see that they include support for wayland, would it be wrong to add it?

Reply:
Well, the application itself puts a window in a specific position which is not possible with Wayland.